### PR TITLE
Add highlights section to README

### PR DESCRIPTION
**Pull request by felipecrs**
This should help new users discover what additional real world-functionalities they can expect from this integration.

You can add more rows to that section. :)

Also, it patches some other minor details.

### CODE REVIEW

**Comment by felipecrs** [comment]
@azerty9971 can you please explain what you meant by this?

image

**Comment by azerty9971** [comment]
> @azerty9971 can you please explain what do you mean by this? As far as I know, the official Tuya integration supports multiple accounts. Doesn't it?
> 
> image

It does, but last time I tried you would not get the devices from all the account being added since each configuration entry resets all the devices at each load. So basically, you can add the config entries but only the last loaded one will have its devices...

**Comment by felipecrs** [comment]
Got it. I'll keep the table as it is then.

File: README.md [hunk 1]
```diff
@@ -4,32 +4,36 @@ Extended Tuya integration for Home Assistant.
...
+The reason why this is not merged in the official Tuya integration is because the way this is done is not officially supported by Home Assistant core team (i.e. this integration uses _hacks_ to do its job).
 
-### Comparison
-
-The following table compares the features of this integration with the official one, as well as the different modes this integration supports.
+### Highlights
 
-Legend:
+- Adds entities needed by the Energy dashboard
+- Adds support for locks (requires [additional configuration here](./docs/configure_locks.md))
```
Comment (felipecrs) [inline README.md:14]:
```suggestion
- Adds support for locks (requires [additional configuration](./docs/configure_locks.md))
```

**Comment by azerty9971** [comment]
Thanks for this <3

**Pull request merged**